js/app.js

```diff
@@ -97,116 +97,143 @@ function renderQuery( query ) {
 }
 
 function pick_map(name){
-	all = `
```

Comment:
dont modify lines unless *mandatory*.

you can add a `switch` without changing `all`.